pkgs/development/python-modules/pysma/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, aiohttp
+, attrs
+, buildPythonPackage
+, fetchPypi
+, jmespath
+, async-timeout
+}:
+
+buildPythonPackage rec {
+  pname = "pysma";
+  version = "0.3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1awcsbk14i2aw01f7b7hrmpn9q6vr9v6la0i9n7ldv1h8rzq6j16";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    async-timeout
+    attrs
+    jmespath
+  ];
+
+  # pypi does not contain tests and GitHub archive not available
```

Comment:
We could fetch the commit which bumped the version but I am not sure if this is the right commit.

Reply:
Issue was opened -> https://github.com/kellerza/pysma/issues/48